packages/helpwave_widget/lib/src/loading/load_error_widget.dart

```diff
@@ -32,7 +32,7 @@ class LoadErrorWidget extends StatelessWidget {
             color: iconColor,
           ),
           const SizedBox(height: distanceBig),
-          Text(errorText),
+          Text(errorText ?? "Error on Load"),
```

Comment:
We should provide translations for such general error messages.

Reply:
Currently, the localization package isn't required in the widget package, therefore we have no access to it, do we want to change this?

Reply:
Let's discuss that tomorrow, okay?